GitUI/UserControls/BlameControl.cs

```diff
@@ -502,7 +502,7 @@ private void ActiveTextAreaControlDoubleClick(object sender, EventArgs e)
             if (_revGrid is not null)
             {
                 _clickedBlameLine = _lastBlameLine;
-                _revGrid.SetSelectedRevision(_lastBlameLine.Commit.ObjectId);
+                _revGrid.SetSelectedRevision(_lastBlameLine.Commit.ObjectId, showNoRevisionMsg: true);
```

Comment:
If a blame is not visible in the grid, the "Blame previous revision" and "Blame this revision" are disabled, so the popup should be redundant.